mmv1/templates/terraform/examples/healthcare_pipeline_job_backfill.tf.tmpl

```diff
@@ -3,11 +3,72 @@ resource "google_healthcare_pipeline_job" "{{$.PrimaryResourceId}}" {
   location = "us-central1"
   dataset = google_healthcare_dataset.dataset.id
   backfill_pipeline_job {
-    mapping_pipeline_job = "${google_healthcare_dataset.dataset.id}/pipelinejobs/{{index $.Vars "mapping_pipeline_name"}}"
```

Comment:
`pipelinejobs` should be `pipelineJobs`. 

I use id here to replace the whole string.

After this fix, a new error comes up. I will stop here.

Reply:
Oh, it is better to just change it to `pipelineJobs`